composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "autoload": {
+        "classmap": [ "vendor/mozilla/wp-sw-manager/" ]
```

Comment:
Yes, I will switch that class to the PSR-4 standard, for now I'm updating the other two classes we're using.

Thank you for the help by the way!